csearch.py

```diff
@@ -142,6 +144,16 @@ def on_finished(self, output, err=None):
         sublime.set_timeout(
             functools.partial(self._finish, output, matches, err=err))
 
+def fix_windows_output(output):
+    result = []
```

Comment:
please add a doc here - "windows" got confusing as I know we're talking OS here, but Windows are also a Sublime context.